draft-ietf-httpbis-messaging-latest.xml

```diff
@@ -900,7 +900,7 @@ https://www.example.org
    Messages are parsed using a generic algorithm, independent of the
    individual field names. The contents within a given field line value are
    not parsed until a later stage of message interpretation (usually after the
-   message's entire header section has been processed).
+   message's entire field section has been processed).
```

Comment:
```suggestion
   message's entire header field section has been processed).
```

Reply:
I think this should be true for bidders and trailers as well.

Reply:
ok.